spec.html

```diff
@@ -6163,7 +6163,7 @@ <h1>CreateRealm ( )</h1>
       <p>The abstract operation CreateRealm with no arguments performs the following steps:</p>
       <emu-alg>
         1. Let _realmRec_ be a new Realm Record.
-        1. Perform CreateIntrinsics(_realmRec_).
+        1. If the host requires use of an existing Realm Record's intrinsics to serve as _realmRec_'s intrinsics, let _realmRec_.[[Intrinsics]] be assigned in an implementation-defined manner to an existing Realm Record's intrinsics record. Otherwise, perform CreateIntrinsics(_realmRec_).
```

Comment:
I believe this change is not necessary since we do not plan to reuse intrinsics between realm records. Mostly because this was meant to solve the identity discontinuity issue that can be solved via a membrane, or the upcoming parameterized evaluator (which we don't know yet how that will work).

Reply:
@caridy The identity discontinuity issue is not solved by the membrane. It is hidden well enough for some practical purposes but not others.

The parameterized evaluator is exactly where this pull request is relevant. We all agreed to move that portion of the realms proposal into the SES / parameterized evaluator proposal. But the new understanding needs new names. The thing we're talking about should no longer be called a Realm record, because it is no longer per Realm. It is per evaluation context.

Attn @jfparadis @michaelfig